inconsistencies.md

```diff
@@ -0,0 +1,153 @@
+# Solving Cassandra consistency
+
+This plan makes the following assumption:
+
+  - We are not willing to rewrite mailbox/cassandra with event sourcing
+  - We will rather identify projections and how to heal them
```

Comment:
what does "identify projections" mean?

Reply:
I adopted a broad definition: "data is duplicated across 2 tables"

Reply:
I can't understand the sentence: "instead of doing ES we will rather identify data that are duplicated accross at least 2 tables and how to heal them"

You probably mean "try to identify possibly inconsistent duplicated information"? Don't you?